packages/fuel-indexer-database/postgres/src/lib.rs

```diff
@@ -834,3 +832,31 @@ pub async fn get_nonce(
 
     Ok(Nonce { uid, expiry })
 }
+
+pub async fn remove_latest_assets_for_indexer(
```

Comment:
This is unused, but want to leave it as this action (removing all latest assets for an indexer) can be valuable